### PR TITLE
Add support bypassing force conversion from other formats (.ass, .ssa etc.) to srt when downloading subtitles

### PR DESCRIPTION
**Pull request by yueyingjuesha**
test

NOTE:
- Currently this option only works for manual download
- I will open a second PR that applies this option to all auto searches and auto-download.
- Since bazarr forbids deleting subtitle other than srt format, you will need to manually delete those non srt subtitle from the disk. Deleting from bazzar web UI won't work for those nonsrt subtitles. I would possibly open a followup PR for addressing the issue

### CODE REVIEW

**Comment by morpheus65535** [comment]
Make sure your PR only includes commit from yourself and not others from the master branch.

File: libs/subliminal_patch/subtitle.py [hunk 1]
```diff
@@ -61,8 +61,10 @@ class Subtitle(Subtitle_):
     pack_data = None
     _guessed_encoding = None
     _is_valid = False
+    use_original_format = False
+    format = "srt" # default format is srt
 
-    def __init__(self, language, hearing_impaired=False, page_link=None, encoding=None, mods=None):
+    def __init__(self, language, hearing_impaired=False, page_link=None, encoding=None, mods=None, original_format=False):
```
Comment (morpheus65535) [inline libs/subliminal_patch/subtitle.py:67]: Really nice parameter but you don't set it to True anywhere... ;-)

Comment (yueyingjuesha) [reply libs/subliminal_patch/subtitle.py:67]: Yeah I am working on front end and kinda of learning how react work.. this pr is still WIP

Comment (morpheus65535) [reply libs/subliminal_patch/subtitle.py:67]: Roger I'll wait for your final version. Thanks!

**Comment by morpheus65535** [comment]
@yueyingjuesha Is this still in your plan to work on that or should I just close this PR?

**Comment by yueyingjuesha** [comment]
@morpheus65535 hey the PR is ready

**Comment by morpheus65535** [comment]
@yueyingjuesha I would be more confortable having all the standing issues solved in the same PR. Would it be possible to include those fixes prior ro merging this?

@vitiko98 I would like to have your input on this PR. I don't see anything obviously wrong about it but having a second pair of eyes looking into it would help. Thanks!

**Review by morpheus65535 — APPROVED** [review]

**Pull request merged**